app/assets/javascripts/import-progress.js

```diff
@@ -0,0 +1,23 @@
+$(function updateImportProgress() {
```

Comment:
Hmm...coulda sworn I tried this and found it to be a bit broken. Decided to drop it on the basis that the function is pretty simple and I couldn't find a working example in collections-publisher or service manual. Should have checked the publisher app!

Anyway, second time's a charm.